### PR TITLE
Improve explanation of advanced scripts workflow (rebased onto develop)

### PR DESCRIPTION
**Pull request by pwalczysko**
This is the same as gh-1549 but rebased onto develop.

---

Add example of output after starting processor for advanced (non-admin) scripting workflow.
See https://www.openmicroscopy.org/site/support/omero5.3-staging/developers/scripts/advanced.html for buit version.

The motivation is also described in [trello card](https://trello.com/c/81ISVYsi/118-cannot-import-script-via-cli#comment-57b59d9982efadf70e294798).

cc @jburel @will-moore

### CODE REVIEW

**Comment by pwalczysko** [comment]
--rebased-from #1549

**Comment by hflynn** [comment]
Building on staging now

**Comment by hflynn** [comment]
@pwalczysko since you have this open anyway, can you get rid of the OMERO.server-Beta-4.2 references in the first section too please?

File: omero/developers/scripts/advanced.txt [hunk 1]
```diff
@@ -46,18 +46,29 @@ openmicroscopy.org.uk and user: will
...
 collaborative group you need to set up your local user processor to
 accept scripts from that user. First, find the ID of the user, then
 start the user processor and give it the user's ID:
 
 ::
 
-    $ cd Desktop/OMERO.server-Beta-4.2/
+    $ cd Desktop/OMERO.server-5.3.x-icexx-bxx/
     $ bin/omero -s openmicroscopy.org.uk -u will user list
     $ bin/omero -s openmicroscopy.org.uk -u will script serve user=5
```
Comment (will-moore) [inline omero/developers/scripts/advanced.txt]: Not part of this PR, but you could remove the "-s openmicroscopy.org.uk -u will" from these commands, since you'll be logged in already.

**Comment by will-moore** [comment]
Looks fine. I haven't tested this workflow recently but it reads fine.
I wonder if we want to update the diagram at the start, since this one still has the (very) old Insight logo.
Also maybe "...need to download 'Ice' " sounds a bit wrong. Probably better to use 'installl Ice'.

**Comment by hflynn** [comment]
@will-moore is the diagram useful? Are there any other changes you'd suggest or shall I just update the logo?

**Comment by pwalczysko** [comment]
This should fix the "text changing" comments of @will-moore

**Comment by will-moore** [comment]
@hflynn I always like a diagram, even if it's very simple.
It's possible there's more detail that could be added to the diagram, but I think even as it is it's fine.

**Comment by will-moore** [comment]
Looks good, thanks. Good to merge.

**Comment by hflynn** [comment]
--rebased-to #1564
